packages/fx-core/tests/core/other.test.ts

```diff
@@ -3,62 +3,82 @@
 
 import "mocha";
 import { FuncValidation, Inputs, Platform, Stage } from "@microsoft/teamsfx-api";
-import {QuestionAppName} from "../../src/core/question";
-import { assert } from "chai"; 
+import { QuestionAppName } from "../../src/core/question";
```

Comment:
Some content is formatted automatically, you can only focus on the `tools: isFeatureFlagEnabled ...` test cases at the end of this file